### DISCUSSION TITLE
'eventlet' has no attribute 'wsgi' when building an exe

### DISCUSSION THREAD

**Discussion by Jey0707** [miguelgrinberg/python-socketio#993] [Q&A] [answered]
I'm having this error while starting an exe made with auto-py-to-exe `AttributeError: module 'eventlet' has no attribute 'wsgi'`

I already had imported eventlet in hidden imports

**Comment by miguelgrinberg** [comment, marked as answer]
Please don't write issues on this package when you have a question or a problem with a third party dependency. The discussion board is the right place to ask.

I don't really have experience with exe generation, but maybe you need to also include `eventlet.wsgi`?

**Comment by Jey0707** [comment]
Solved, in normal python script you don't need to import `from eventlet import wsgi`, but if you want to make an exe, you must import it.
